vault-plugins/flant_iam/extensions/extension_server_access/path_server.go

```diff
@@ -253,7 +258,7 @@ func (b *serverBackend) handleRegister() framework.OperationFunc {
 		}
 
 		resp := &logical.Response{Data: map[string]interface{}{
-			"multipassJWT": jwt,
+			"multipassJWT": jwtToken,
 			"uuid":         serverUUID,
```

Comment:
I suggest we use the similar naming in contract. It is called "token" in SA multipass 

https://github.com/flant/negentropy/pull/171/files#diff-d0dfbf3f86ec78f37d9968dcf6c3e0664f3af16baa3b2cd027029a301797b619R606

Reply:
@savelyev-an FYI

I will not fix it because I did't write extension_server_access. It need to change in authd (maybe). But I agree with 
Eugene